src/backend/src/common/constants/gov-ministries.constant.ts

```diff
@@ -3,77 +3,102 @@ import { GovMinistriesEnum } from '../enums/gov-ministries.enum';
 export const GovMinistries = {
   [GovMinistriesEnum.ADVANCED_EDUCATION_AND_SKILLS_TRAINING]: {
     label: 'Advanced Education and Skills Training',
+    code: 'AEST',
```

Comment:
This ministry is now: Post-Secondary Education and Future Skills, acronym: 'PSEF'(?)